src/server/server_family.cc

```diff
@@ -61,7 +61,8 @@ ABSL_FLAG(string, requirepass, "",
           "If empty can also be set with DFLY_PASSWORD environment variable.");
 ABSL_FLAG(string, save_schedule, "",
           "glob spec for the UTC time to save a snapshot which matches HH:MM 24h time");
-ABSL_FLAG(string, snapshot_cron, "", "cron expression for the time to save a snapshot");
+ABSL_FLAG(string, snapshot_cron, "",
+          "cron expression for the time to save a snapshot, crontab style");
```

Comment:
Perhaps it's worth explicitly specifying the fields we expect?

Reply:
I thought about just including the column names, and indeed refer to crontab like you did for more details. But perhaps it's not really helpful.